contracts/RiskManagerV1.sol

```diff
@@ -461,6 +652,13 @@ contract RiskManagerV1 is IRiskManager, Auctioneer, Ownable {
         );
     }
 
+    /// @return Total supply of the COV token.
+    function covTotalSupply() internal view returns (uint256) {
+        UnderwriterToken underwriterToken =
+            AssetPool(coveragePool.assetPool()).underwriterToken();
+        return underwriterToken.totalSupply();
+    }
```

Comment:
It feels that underwriter token calculations are not something the risk manager should care about. We could introduce function similar to `amountToSeize` in `CoveragePool` to compute the amount of underwriter tokens one should receive based on the portion of the pool to grant. 

We do have `amountToSeize(uint256 portionToSeize)` as an amount of collateral tokens to seize and we could have next to it `amountToGrant(uint256 portionToGrant)` as an amount of underwriter tokens to grant. What do you think?

Reply:
Ah, I see it now. You need the amount because the reward is either fixed amount or % and we need to calculate this %.

Reply:
On the other hand, given that all functions are internal to the reward manager, wouldn't it be more elegant:
```
    function getLiquidationNotifierReward(
        Storage storage self,
        CoveragePool coveragePool
    ) internal view returns (uint256) {
        if (self.liquidationNotifierRewardAmount > 0) {
            return self.liquidationNotifierRewardAmount;
        }

        return coveragePool.amountToGrant(self.liquidationNotifierRewardPercentage);
    }
```
?

Reply:
Sounds good. See https://github.com/keep-network/coverage-pools/pull/111/commits/3343bab820f506e1d1df2464cd601e19995dd516. I just named it `covAmountToGrant` to not confuse with `amountToSeize` which is about collateral tokens.